CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```

Comment:
maybe `... ,sexual identity, or sexual orientation` this way it covers both and not just the two together.

Reply:
What exactly is the difference between "sex characteristics, gender identity and expression", and  "sexual identity, or sexual orientation" ? To me they describe the same thing. However, it probably doesn't hurt to have more inclusions, I'm just curious...

Reply:
maybe add something like "linguistic expression"? We don't want people to be harassed justed because they can't write proper english either, right?

Reply:
from google about gender vs sex: In general terms, "sex" refers to the biological differences between males and females, such as the genitalia and genetic differences. "Gender" is more difficult to define, but it can refer to the role of a male or female in society, known as a gender role, or an individual's concept of themselves, or gender identity.

i think that listing everything is just a way to cover all the possible bases. 

with regards to linguistic expression, i dont think that we should add something like that. If required i think that it would fall under education. I am also wary about adding something like that as if someone was very sensitive about it they might feel attacked for some typos which had occurred in a doc string. Obviously, if the comments about the typos constitute harassment or abuse then this, in my belief, would be covered by the education section of this paragraph.